examples/nodejs/cache/advanced.ts

```diff
@@ -91,8 +91,6 @@ async function setGetDeleteExample() {
     logger.info('cache miss');
   } else if (getResponse instanceof CacheGet.Error) {
     logger.info(`Error: ${getResponse.message()}`);
-  } else {
```

Comment:
with this new pattern, this now errors out since we have exhausted all type possibilities, and the `getResponse` type is not of type `never`

![image (8)](https://github.com/momentohq/client-sdk-javascript/assets/11823378/b187ffba-a102-4f8e-bf2e-650029a84027)